frontend/pages/doctor/appointments-overview.tsx

```diff
@@ -0,0 +1,132 @@
+import AppointmentReminders from "@frontend/components/doctor/appointment-reminders";
+import moment, { Moment } from "moment";
+import { MinusIcon } from "@chakra-ui/icons";
+import AppointmentCalendar from "@frontend/components/doctor/appointment-calendar";
+import { PatientBasicInformation } from "@frontend/models/patient";
+import { Scrollbars } from "react-custom-scrollbars-2";
+
+export async function getServerSideProps(context: any) {
+    const session = await getSession(context);
+    const userId = session?.user.AccountId;
+    let appointmentAndPatientList: { appointmentList: Appointment[]; patientList: PatientBasicInformation[] } = {
+        appointmentList: [],
+        patientList: [],
+    };
+    try {
+        if (session?.user.Role === USER_ROLES.doctor) {
+            const appointmentListResponse: any = await fetch(
+                serverURL + "/doctors/getAppointmentsAndPatients/" + userId
+            );
+            appointmentAndPatientList = await appointmentListResponse.json();
+        }
+    } catch {}
+    return {
+        props: {
+            session,
+            appointmentList: appointmentAndPatientList.appointmentList,
+            patientList: appointmentAndPatientList.patientList,
+            userId,
+        },
+    };
+}
+export default function AppointmentsOverview({
+    appointmentList,
+    patientList,
+    userId,
+}: {
+    appointmentList: Appointment[];
+    patientList: PatientBasicInformation[];
+    userId: number;
+}) {
+    const { onOpen, isOpen, onClose } = useDisclosure();
+
+    function sortAppointments(section: string): Appointment[] {
+        const upcomingAppointments: Appointment[] = [];
+        const pastAppointments: Appointment[] = [];
+        let appointmentDate: Moment = moment();
+
+        appointmentList.map((appointment: Appointment) => {
+            appointmentDate = moment(
+                `${appointment.date} ${appointment.time.substring(0, appointment.time.indexOf(" "))}`,
+                "YYYY-MM-DD HH:mm"
+            );
+            if (appointmentDate.isBefore()) {
+                pastAppointments.push(appointment);
+            } else if (appointmentDate.isAfter()) {
+                upcomingAppointments.push(appointment);
+            }
+        });
+        if (section === SECTION.UPCOMING) return upcomingAppointments;
+        else if (section === SECTION.PAST) return pastAppointments;
+        return appointmentList;
+    }
+    return (
+        <Box m={10}>
+            <Heading>Appointments</Heading>
+            <Flex flexDirection={{ base: "column", sm: "column", md: "column", lg: "row" }}>
+                {/* Calendar */}
+                <Box display={{ base: "none", md: "none", lg: "none", xl: "block", sm: "none" }} flex="1">
+                    <AppointmentCalendar appointmentList={appointmentList} />
+                </Box>
+                {/* Legend */}
+                <Box flex="1" h={650} minW={450}>
```

Comment:
```suggestion
                <Box flex="1" h={650} minW={300}>
```
Should change to 300 to not brake the layout on mobile